app/controllers/tag_wranglings_controller.rb

```diff
@@ -8,7 +8,10 @@ class TagWranglingsController < ApplicationController
 
   def index
     @counts = tag_counts_per_category
-    unless params[:show].blank?
+    if params[:show].blank?
+      authorize :wrangling, :full_access? if logged_in_as_admin?
+    else
+      authorize :wrangling, :read_access? if logged_in_as_admin?
```

Comment:
And then based on hiding the option from the navbar but not the sidebar, I think wrangling tools should be visible to read_access admins. PAC doesnt care either way and it's the only way they'd be able navigate to the mass bins.